src/DotNet/Writer/MethodBodyWriterBase.cs

```diff
@@ -12,6 +12,8 @@ public abstract class MethodBodyWriterBase {
 		protected IList<Instruction> instructions;
 		/// <summary/>
 		protected IList<ExceptionHandler> exceptionHandlers;
+		/// <summary>The module context</summary>
+		protected readonly ModuleContext context;
```

Comment:
Does this need to be protected?

Also no code uses it so it seems like it's not needed.